stylesheets/components/grid.css

```diff
@@ -0,0 +1,21 @@
+.grid {
+    display: grid;
+    max-width: fit-content;
+    margin: 2rem;
+}
+
+.grid-2 {
+    grid-template-columns:  repeat(2, 1fr);
+    gap: 1rem;
+}
+
+.grid-3 {
+    grid-template-columns: repeat(3, 1fr);
+    gap: 1.5rem;
+}
+
+.grid-item {
+    border: 1px solid var(--black);
```

Comment:
Do not write this border here in the component. You may want to use this grid component without it. And for showcasing the grid component on your CL page, you can add this border in the CSS file meant for the page.

Reply:
Ok, I'll do that!